### PR TITLE
Added Scala wrapper for calling CarRideStats python script

### PR DESCRIPTION
**Pull request by carloscaldas**
Resolves: https://github.com/LBNL-UCB-STI/beam/issues/2457

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2468)

### CODE REVIEW

File: src/main/scala/beam/analysis/AnalysisProcessor.scala [hunk 1]
```diff
@@ -53,39 +54,14 @@ object AnalysisProcessor extends ExponentialLazyLogging {
         }
       )
       logger.info(s"Running python script: $scriptPath with args $args")
-      PythonProcess(Some((Seq("python", scriptPath) ++ args).mkString(" ").run(processLogger)))
+      val str = (Seq("python3", scriptPath) ++ args).mkString(" ")
```
Comment (JustinPihony) [inline src/main/scala/beam/analysis/AnalysisProcessor.scala]: This should be `processName`, right?

Comment (carloscaldas) [reply src/main/scala/beam/analysis/AnalysisProcessor.scala]: right. thx

File: src/main/scala/beam/analysis/carridestats/CarRideStatsAreaBoundBox.scala [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package beam.analysis.carridestats
+
+case class CarRideStatsPoint(x: Double, y: Double) {
+  override def toString: String = s"($x,$y)"
+}
+
+case class CarRideStatsFilterAreaBoundBox(topLeft: CarRideStatsPoint, bottomRight: CarRideStatsPoint)
```
Comment (REASY) [inline src/main/scala/beam/analysis/carridestats/CarRideStatsAreaBoundBox.scala]: I think you don't need all of these. There is already a usage of so called `Envelope` type from jts. The usage can be found, for example, in ZonalParkingManager: https://github.com/LBNL-UCB-STI/beam/blob/0dd3a931369a53f8b1930daf4213a9592783042e/src/main/scala/beam/agentsim/infrastructure/ZonalParkingManager.scala#L254

File: src/main/python/calibration/generate_rides_with_google_maps.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (REASY) [inline src/main/python/calibration/generate_rides_with_google_maps.py]: How can we be sure that use has Python3 and all the needed dependencies? Should we have `requirement.txt` for `pip` and make sure users install it before running Beam?

**Pull request merged**